models/migrations/v1_19/v232.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package v1_19 //nolint
+
+import (
+	auth_models "code.gitea.io/gitea/models/auth"
+	"code.gitea.io/gitea/modules/timeutil"
+
+	"xorm.io/xorm"
+)
+
+func AddScopeForAccessTokens(x *xorm.Engine) error {
+	type AccessTokenWithDefaultScope struct {
+		ID             int64 `xorm:"pk autoincr"`
+		UID            int64 `xorm:"INDEX"`
+		Name           string
+		Token          string `xorm:"-"`
+		TokenHash      string `xorm:"UNIQUE"` // sha256 of token
+		TokenSalt      string
+		TokenLastEight string                       `xorm:"token_last_eight"`
+		Scope          auth_models.AccessTokenScope `xorm:"NOT NULL DEFAULT 'all'"`
```

Comment:
Copy the define here is better because we don't know if the definition will be changed in future.

Reply:
Solved